### PR TITLE
CAF-2846: Updated readme external parameters to include logging envir…

### PR DESCRIPTION
**Pull request by arogan92**
…onment variable.

### CODE REVIEW

**Comment by glucy** [comment]
@mulhollandc Can you please review this for Adam.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,11 @@ The following parameters may be set:
         <td>25080</td>
         <td>This is the port that the Audit Web Service is configured to listen on.</td>
       </tr>
+	  <tr>
+		<td>CAF_LOG_LEVEL</td>
+		<td>INFO</td>
```
Comment (mulhollandc) [inline README.md:94]: Should this not be 'info'? i.e. lowercase as you mentioned in the container README this is case sensitive.

File: README.md [hunk 2]
```diff
@@ -89,6 +89,11 @@ The following parameters may be set:
...
+		<td>This environment variable can be used to override the current default logging level in Audit Web Service. The current default level is INFO.</td>
```
Comment (mulhollandc) [inline README.md]: I would change this to something like 'The logging level for the Audit Web Service. The supported logging levels include warn, error, info, trace, debug or all.'.

**Review by mulhollandc — CHANGES_REQUESTED** [review]

**Comment by arogan92** [comment]
README changes made after comments

**Comment by arogan92** [comment]
Could you approve this pull request also @mulhollandc ?

**Review by mulhollandc — APPROVED** [review]

**Pull request merged**